astropy/wcs/utils.py

```diff
@@ -31,6 +39,22 @@
 ]
 
 
+solar_system_obj_dict = {
```

Comment:
I believe global "constants"'s names should be capitalized, see https://peps.python.org/pep-0008/#constants